src/utils/clmfates_interfaceMod.F90

```diff
@@ -1036,9 +1039,13 @@ subroutine init(this, bounds_proc, flandusepftdat)
 
       call this%init_history_io(bounds_proc)
 
-      ! Report Fates Parameters (debug flag in lower level routines)
-      call FatesReportParameters(masterproc)
-
+      ! Check through FATES parameters
+      ! THis step is performed after the parameter
+      ! reads because we need to cross-reference
+      ! these against namelist settings
+      !call FatesCheckParameters(masterproc)
```

Comment:
should remove this check, this is now called on the FATES side: https://github.com/NGEET/fates/pull/1493/files#diff-a2aa2071b57fc11ed7acd14061cab23ff92b700f208ee30700754a5a6709c8bdR1093